src/domain/session/toast/CallToastNotificationViewModel.ts

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import type {GroupCall} from "../../../matrix/calls/group/GroupCall";
+import type {Room} from "../../../matrix/room/Room.js";
+import {IAvatarContract, avatarInitials, getIdentifierColorNumber, getAvatarHttpUrl} from "../../avatar"; 
+import {LocalMedia} from "../../../matrix/calls/LocalMedia";
+import {BaseClassOptions, BaseToastNotificationViewModel} from "./BaseToastNotificationViewModel";
+import {SegmentType} from "../../navigation";
+
+type Options<N extends MinimumNeededSegmentType = SegmentType> = {
+    call: GroupCall;
+    room: Room;
+} & BaseClassOptions<N>;
+
+// Since we access the room segment below, the segment type
+// needs to at least contain the room segment!
+type MinimumNeededSegmentType = {
+    "room": string;
+};
+
+export class CallToastNotificationViewModel<N extends MinimumNeededSegmentType = SegmentType, O extends Options<N> = Options<N>> extends BaseToastNotificationViewModel<N, O> implements IAvatarContract {
+    constructor(options: O) {
+        super(options);
+        this.track(
+            this.call.members.subscribe({
```

Comment:
Could you make an ObservableValue class that returns the size? Then I could use it in my PR as well :)